saltproc/version.py

```diff
@@ -26,50 +26,62 @@
                "Topic :: Scientific/Engineering"]
 
 # Description should be a one-liner:
-description = "saltproc: a template for small scientific Python projects"
+description = "saltproc: the on-line reprocessing driver for SERPENT"
 # Long description will go up on the pypi page
 long_description = """
-
 Shablona
 ========
-Shablona is a template project for small scientific Python projects.
+Saltproc is a on-line reprocessing driver for SERPENT, where it 
+runs SEPRENT from a given input file and parameters and performs
+user-defined processing of the fuel salt by adding or removing
+waste, fissile, and fertile streams.
+License
+=======
+The 3-Clause BSD License
 
-It contains software implementations of an analysis of some simple data, but
-more importantly, it contains infrastructure for testing, documentation,
-continuous integration and deployment, which can be easily adapted
-to use in other projects.
+Copyright (c) 2018, Andrei Rykhlevskii
+All rights reserved.
 
-To get started using these components in your own software, please go to the
-repository README_.
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
 
-.. _README: https://github.com/uwescience/saltproc/blob/master/README.md
+* Redistributions of source code must retain the above copyright notice, this
+  list of conditions and the following disclaimer.
 
-License
-=======
-``saltproc`` is licensed under the terms of the MIT license. See the file
-"LICENSE" for information on the history of this software, terms & conditions
-for usage, and a DISCLAIMER OF ALL WARRANTIES.
+* Redistributions in binary form must reproduce the above copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
 
-All trademarks referenced herein are property of their respective holders.
+* Neither the name of default-bsd-3-clause nor the names of its
+  contributors may be used to endorse or promote products derived from
+  this software without specific prior written permission.
 
-Copyright (c) 2015--, Ariel Rokem, The University of Washington
```

Comment:
We should keep a license file called LICENSE_shablona which credits ariel for the shablona backbone of this work. THat's the point of an attirbution license. You don't just delete it.